### DISCUSSION TITLE
Row count exceeded 100,000?

### DISCUSSION THREAD

**Discussion by svetlasyrimis** [planetscale/discussion#190] [Q&A] [answered]
Hi there, new to Planet Scale and MySQL.

Trying to export a large table from my sandbox Planet Scale db with mysqldump and I get that message. Also when trying to run any SQL statement on more than 100,000 rows. Does it have to do with the fact that I am currently on the free developer plan?

**Comment by gnumoreno** [comment, marked as answer]
Hey @svetlasyrimis 

Have you checked the pscale cli?

https://docs.planetscale.com/reference/database

You can use `pscale database dump <DB_NAME> <BRANCH_NAME>`

But to get over the 100k rows limitation you can also use:

`mysql> set workload=olap;`

**Reply by pencilcheck** [reply]
I tried it on planetscale console, it didn't work for me.

Found another discussion talking about it: https://github.com/planetscale/discussion/discussions/388

**Comment by svetlasyrimis** [comment]
Hey @gnumoreno 

Thank you so much for your quick reply!
Couldn't get `set workload=olap;` to work but `pscale database dump <DB_NAME> <BRANCH_NAME>` did the trick

**Reply by svetlasyrimis** [reply]
@gnumoreno would `set workload=olap;` help with inserts higher than 100,000(per table)?

**Reply by gnumoreno** [reply]
I'm not sure. I think it just affect reads. I will wait for confirmation from someone in this repo.

**Reply by jgreet** [reply]
It might help if in a single INSERT statement you were trying to add more than 100,000 rows, or if that single INSERT was taking longer than 900 seconds.

**Reply by svetlasyrimis** [reply]
@jgreet yeah that's the case, single insert statement will fail with more than 100 000 rows

So my assumption is that it will succeed if I have two consecutive calls each trying to insert not more than 
100, 000 rows. Or is there a rate limit? 
Don't remember if I tried that.

**Reply by jgreet** [reply]
Right, that should be OK. There isn't an artificial rate limit on rows inserted per time period.

**Comment by pencilcheck** [comment]
How do I do it with truncate table?? I tried `DELETE FROM [table name];` from my prisma.$executeRaw.

Found another discussion talking about it: https://github.com/planetscale/discussion/discussions/388

**Reply by orware** [reply]
Hello @pencilcheck,

With the recent additions of our ability to turn on/off the "safe migrations" feature as detailed here in our documentation:
https://planetscale.com/docs/concepts/safe-migrations#protection-against-accidental-schema-changes

You have a path forward to using `TRUNCATE` on a table with safe migrations disabled temporarily. 

I hope this is helpful!

**Comment by SriramItrend** [comment]
Same issue i have faced 
but different scenario ,
if trying to add more than 1,00,000 rows in my table , throw the error 

"
Mysql:target:itrenddb.-.primary:vttablet:rpc error :code = Aborted desc = Row Count exceeded 100000 (CallerID: 1vcsx0eo349iqhfcw4c0)

"


Please be suggest how to solve
